### DISCUSSION TITLE
No one except me gets emails using Gmail

### DISCUSSION THREAD

**Discussion by NickHodges** [emailjs-com/help#51] [Q&A]
Hey --

I am using emailjs to send an email for people to get a free copy of my mom's book by way of trying to seed reviews.  

I have an Astro site that sends the emails when a person signs up.  It was a good learning experience, and I got it working...sort of.

The template uses my gmail account to send the emails.  The emails get sent according to my dashboard, but I appear to be the only one that receives anything.  I get every email every time I send one to my gmail.com address, but none of my friends that are helping me test ever get the emails.  I tried my vanity domain email, and that didn't work either.  I never got that email.

Am I missing something?  Will GMail do what I need?  

Any help will be gratefully received.  Happy to provide more information if it is useful.

**Comment by xr0master** [comment]
Please ask your friends to check their SPAM folder. If your emails are there, then I recommend you work on the email content. Let me know if this is your case, and I'll give you more information.

**Reply by NickHodges** [reply]
I did have everyone look in their Spam folder, and the email wasn't there.  I checked my own spam folder on my non-gmail account and it wasn't there either. 

So, no, they aren't there at all.  

I can only receive email on my personal gmail account. Some of the people that tried used gmail accounts and didn't get the email.

**Reply by NickHodges** [reply]
Okay, so I think I might have figured out what is going on --- 

I am getting the email on my gmail account!

In fact, I  just verified that that is exactly what is happening.  

So I'm doing something wrong in my code.  :-)

Thanks for the response, I appreciate it.

**Reply by xr0master** [reply]
To change the recipient's address, you may visit your template in the EmailJS dashboard and replace a value of the "To Email" field. You can also use a dynamic variable for this field.

**Reply by greghawk** [reply]
@xr0master So the only way I ever get an email actually to receive is from here.


<img width="578" alt="image" src="https://github.com/user-attachments/assets/a79d4247-5164-40df-a1a7-43623a1737a3" />


My google account is setup to "Send email on your behalf". However anytime i try to use the email service to send it says everything is good but no email is received at the To mailbox. Not in spam. Tried different emails.  Tried MXToolbox. Only thing it noted was SPF record but if that were the case how would your test email work?

**Reply by xr0master** [reply]
The information shown in your screenshot refers to the provider used to send emails FROM the address. It has no connection to the recipient’s email address, which is configured in the template.

**Reply by greghawk** [reply]
@xr0master I am writing the cause in here so others may know.  So my google account that is connected as the email service has aliases on it. The To address is one of my aliases.  When you do this and have an iphone or mac either google or apple marks these messages as read and moves them to an archive folder on ios and the "all mail" folder within gmail. This effectively makes it seem gone as it removes the inbox flag on gmail.
